GetSchwifty/Code Helper/FormattingHelper.swift

```diff
@@ -203,6 +203,7 @@ private extension UITextView {
     
     // MARK: - ACTIONS
     func newLine(_ times: UInt = 1) {
+        guard times > 0 else { return }
```

Comment:
I'm not sure what you mean. We can as well change times to an Int. In either case, if a 0 is passed, it would crash because the range in the next line would be 1...0. So we need to make sure times is greater than 0.